### PR TITLE
fix(homepage): typo corrections and updated twitter handles

### PR DESCRIPTION
**Pull request by goffxnca**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation
I was browsing the [Docusaurus](https://docusaurus.io/) homepage and spotted a few typos. so, I cloned the repo to fix those, and while I was at it, I noticed a few other things to update like switching links to https and refreshing twitter handles.

## Test Plan
`-`

### Test links
`-`

## Related issues/PRs
`-`

### CODE REVIEW

File: website/src/data/tweets.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ const TWEETS: TweetItem[] = [
       </>
     ),
     showOnHomepage: true,
-    githubUsername: 'markerikson',
+    githubUsername: 'acemarke',
```
Comment (slorber) [inline website/src/data/tweets.tsx:33]: looks to me the github username was correct: https://github.com/markerikson/

File: website/src/data/tweets.tsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ const TWEETS: TweetItem[] = [
       </>
     ),
     showOnHomepage: true,
-    githubUsername: 'mlynch',
+    githubUsername: 'maxlynch',
```
Comment (slorber) [inline website/src/data/tweets.tsx:61]: same 
https://github.com/mlynch/

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks

the github usernames looks wrong now

We need you to sign the CLA, can't merge without it

**Comment by goffxnca** [comment]
> Thanks
> 
> the github usernames looks wrong now
> 
> We need you to sign the CLA, can't merge without it

CLA signed & reflected thx.

File: website/src/data/tweets.tsx (hunk 1 above)
Comment (goffxnca) [reply website/src/data/tweets.tsx]: my bad, i was confusing with twitter handles, reverted thx! 🙏

File: website/src/data/tweets.tsx (hunk 2 above)
Comment (goffxnca) [reply website/src/data/tweets.tsx]: fixed here too.

**Pull request merged**